Cargo.toml

```diff
@@ -14,6 +14,9 @@ members = [
     "runtime/runtime",
     "runtime/wasm",
     "runtime/wasm/runtest",
+    "runtime/rust-api",
```

Comment:
Should we have this API in separate repo actually?
Because it has nothing to do with the runtime itself

Reply:
I second that